packages/medulas-react-components/src/components/Drawer/index.tsx

```diff
@@ -141,6 +144,11 @@ function PersistentDrawerRight({ children, items, elevation = 0 }: Props): JSX.E
         <List component="nav" id={DRAWER_HTML_ID}>
           {items.map(item => (
             <ListItem button key={item.text}>
+              {item.icon && (
```

Comment:
I believe the `<Divider />` in line 143, introduces an extra line just before the list items, making an ugly effect. Try to remove it.